runtime-modules/working-group/src/lib.rs

```diff
@@ -1488,6 +1534,7 @@ impl<T: Trait<I>, I: Instance> Module<T, I> {
         opening_human_readable_text_constraint: InputValidationLengthConstraint,
         worker_application_human_readable_text_constraint: InputValidationLengthConstraint,
         worker_exit_rationale_text_constraint: InputValidationLengthConstraint,
+        worker_storage_text_constraint: InputValidationLengthConstraint,
```

Comment:
I think this "raw" storage field should be versatile and allow potentially having arbitrary binary data in it, not just plain text. The worker should also be able to set it to an empty value. For this reason I think we should allow the minimum length to be zero. Does `InputValidationLengthConstraint` and validation allow for min length to be 0?

Also `worker_storage_text_constraint` can be re-named to reflect this to maybe: `worker_storage_size_constraint`

What do you think?

cc: @bedeho

Reply:
Nice catch. Yep, `InputValidationLengthConstraint` and validation allows for min length to be 0.

Reply:
`Also worker_storage_text_constraint can be re-named to reflect this to maybe: worker_storage_size_constraint`
Done https://github.com/Joystream/joystream/pull/2180/commits/9f1d0ce17addd360318b008408e4f5516656d6ec